### DISCUSSION TITLE
JSON => HistoryGraph

### DISCUSSION THREAD

**Discussion by AFPass** [Zefau/ioBroker.jarvis#1388] [Ideas / Feature Requests]
Ich habe im ioBroker ein DP der ein JSON enhält.
Bekomme ich das irgendwie als HistoryGraph angezeigt?

**Comment by Zefau** [comment]
HistoryGraph ist nur für History Daten. Ich kann es in das Chart Modul einbauen. Mach gerne ein Feature Request inkl dem JSON auf.
